### DISCUSSION TITLE
Trim isolated pores

### DISCUSSION THREAD

**Discussion by engphb** [PMEAL/OpenPNM#2586] [Q&A] [answered]
Hello professor Gostick, how are you? I have a question about the Snow algorithm. What are considered connected pores after modeling? Are they pores that make contact with the external environment or is a pore connected to another pore?

**Comment by jgostick** [comment, marked as answer]
I believe that you sent me the same question in an email right?  I hope my answer was sufficient

**Reply by engphb** [reply]
Hello professor, our discussion was still open. So, moving on. Our university microCT scan is from the Zeiss company. Zeiss defines tomographic image resolution as 2 voxels. As my voxel size is 12 um with bin 2 (if I use bin 1 the voxel size will be 6 um), the resolution is 24 um. That's a certainty. My question is whether I use the value of 12 um or 24 um in the OpenPNM code.

**Reply by jgostick** [reply]
Hi @engphb, we seem to have different definitions, so let's let get it straight.  If you take a scan of a spherical ball which has a physical radius of 100 um, and in your image the ball is 20 voxels width, then the resolution of that image is 5 um per voxel.  The binning and all other considerations are irrelevant at this point. You just need to tell porespy how much micrometers are represented by each voxel in the image. The `voxel_size` argument in all the porespy functions is just this...how many microns long is a voxel edge.

**Reply by engphb** [reply]
Okay professor, I understand now. Thanks!
